test/rulesets/LinearAlgebra/factorization.jl

```diff
@@ -130,7 +126,7 @@ end
             X = [1.0 2.0; 3.0 4.0; 5.0 6.0]
             F, dX_pullback = rrule(svd, X)
             X̄ = Composite{typeof(F)}(U=zeros(3, 2), S=zeros(2), V=zeros(2, 2))
-            for p in [:U, :S, :V]
+            for p in [:U, :S, :V, :Vt]
                 Y, dF_pullback = rrule(getproperty, F, p)
                 Ȳ = ones(size(Y)...)
```

Comment:
I know it's not part of this PR, but it would be safer to have a random cotangent here.

Reply:
Sure that's fine.

Reply:
Actually, testing with `test_rrule` shows up problems:
```julia
ERROR: MethodError: no method matching adjoint(::Composite{Adjoint{Float64,Array{Float64,2}},NamedTuple{(:parent,),Tuple{Array{Float64,2}}}})
Closest candidates are:
  adjoint(::Missing) at missing.jl:100
  adjoint(::Number) at number.jl:169
  adjoint(::Adjoint) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/LinearAlgebra/src/adjtrans.jl:164
  ...
Stacktrace:
 [1] top-level scope at REPL[32]:1
```
for the
```julia
       elseif x === :V
             C(Vt=Ȳ',)
```
part. See https://github.com/JuliaDiff/ChainRulesCore.jl/issues/312, do you have an opinion on that?